src/pages/Home/component/Title/index.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { StyledHomeTitle } from "./styles";
+
+const index = () => {
+  return (
+    <StyledHomeTitle>
+      <h2>Eisa Rezaei </h2>
+      <h2>React.js Web developer </h2>
+      <h2>based in Iran</h2>
```

Comment:
change this text font to https://fonts.google.com/specimen/Kaisei+HarunoUmi#glyphs. and remove the `based` word.